src/platform/forms/tests/save-in-progress/SaveInProgressIntro.unit.spec.jsx

```diff
@@ -82,6 +82,49 @@ describe('Schemaform <SaveInProgressIntro>', () => {
     ).to.equal('testing');
     tree.unmount();
   });
+  it('should render in progress message with header', () => {
+    const user = {
+      profile: {
+        savedForms: [
+          {
+            form: VA_FORM_IDS.FORM_10_10EZ,
+            metadata: {
+              lastUpdated: 946684800,
+              expiresAt: moment().unix() + 2000,
+            },
+          },
+        ],
+        prefillsAvailable: [],
+      },
+      login: {
+        currentlyLoggedIn: true,
+        loginUrls: {
+          idme: '/mockLoginUrl',
+        },
+      },
+    };
+
+    const tree = shallow(
+      <SaveInProgressIntro
+        saveInProgress={{ formData: {} }}
+        pageList={pageList}
+        formId="1010ez"
+        user={user}
+        fetchInProgressForm={fetchInProgressForm}
+        removeInProgressForm={removeInProgressForm}
+        toggleLoginModal={toggleLoginModal}
+        formConfig={formConfig}
+        headingLevel={1}
+      />,
+    );
+
+    expect(tree.find('.usa-alert').text()).to.contain(
+      'Your application is in progress',
+    );
+    expect(tree.find('h1')).to.exist;
```

Comment:
Can this be taken a step further so that the expectation is that the `<h1>` node has `text()` which reads "Your application is in progress"?

Reply:
Absolutely! https://github.com/department-of-veterans-affairs/vets-website/pull/17437/files#diff-a537e37b0547e6d4831e21f0318d284e75a9a91dc4b2e784f0135a5ae6619d64R121